src/ImageSharp/Formats/Jpeg/Components/Encoder/L8ToYConverter.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Runtime.CompilerServices;
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace SixLabors.ImageSharp.Formats.Jpeg.Components.Encoder
+{
+    /// <summary>
+    /// Provides 8-bit lookup tables for converting from L8 to Y colorspace.
+    /// </summary>
+    internal unsafe struct L8ToYConverter
```

Comment:
There are no lookup tables or any other information to work with during grayscale conversion.

All methods in this type can be just moved into `static` methods within `LuminanceForwardConverter<TPixel>` and the `L8ToYConverter` struct can be removed.

Reply:
I modeled this to be similar to the existing YCbCr classes. But Grayscale conversion is much simpler, so moved it into the LuminanceForwardConverter class.